packages/test/test-end-to-end-tests/src/test/cellEndToEndTests.spec.ts

```diff
@@ -279,6 +280,43 @@ describeNoCompat("SharedCell orderSequentially", (getTestObjectProvider) => {
         containerRuntime = dataObject.context.containerRuntime as ContainerRuntime;
     });
 
+    it("Should rollback set", async () => {
+        let error: Error | undefined;
+        try {
+            containerRuntime.orderSequentially(() => {
+                sharedCell.set(0);
+                throw new Error(errorMessage);
+            });
+        } catch (err) {
+            error = err as Error;
+        }
+
+        assert.notEqual(error, undefined, "No error");
+        assert.equal(error?.message, errorMessage, "Unexpected error message");
+        assert.equal(containerRuntime.disposed, false);
+        assert.equal(sharedCell.get(), undefined);
+        // rollback
```

Comment:
@scarlettjlee Based on our discussion, we are not going to include the previous local values in event listener. Is there any other way to test the order of previous values is correct or the previous value really exists? Like the test for map: https://github.com/microsoft/FluidFramework/blob/ffea7550dd3ed116a75165b99d82486472ce8e2d/packages/test/test-end-to-end-tests/src/test/mapEndToEndTests.spec.ts#L475

Reply:
Watch the events. You should see a valueChanged and a delete.

Reply:
You can tell indirectly that the previous value must have been correct by checking the value after rollback.